src/OrleansVSTools/VSProjectTemplateGrainInterface/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.VSSDK.BuildTools" version="14.2.25201" targetFramework="net451" developmentDependency="true" />
```

Comment:
This doesn't add the package to the unfolded template (for the end users),  right?